src/Razor/test/Microsoft.NET.Sdk.Razor.Test/IntegrationTests/StaticWebAssetsIntegrationTest.cs

```diff
@@ -190,16 +175,15 @@ public async Task Rebuild_Success_RecreatesManifestAndCache()
             }
 
             var path = Assert.FileExists(result, OutputPath, "AppWithPackageAndP2PReference.dll");
-            var assembly = Assert.ContainsEmbeddedResource(path, "Microsoft.AspNetCore.StaticWebAssets.xml");
-            using (var reader = new StreamReader(assembly))
-            {
-                var data = reader.ReadToEnd();
-                Assert.Equal(expectedManifest, data);
-            }
+            var manifestLocation = Assert.FileExists(result, OutputPath, "AppWithPackageAndP2PReference.StaticWebAssets.xml");
+
+            Assert.FileExists(result, OutputPath, "AppWithPackageAndP2PReference.dll");
```

Comment:
Should this be checking `manifestLocation`

Reply:
This is a separate check to ensure the dll is there